src/Analyzers/Core/CodeFixes/UseCoalesceExpression/UseCoalesceExpressionForIfNullStatementCheckCodeFixProvider.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.LanguageService;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CodeFixes.UseCoalesceExpression
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, LanguageNames.VisualBasic, Name = PredefinedCodeFixProviderNames.UseCoalesceExpressionForIfNullStatementCheck), Shared]
+    [ExtensionOrder(Before = PredefinedCodeFixProviderNames.AddBraces)]
```

Comment:
Curious why this is needed.

Reply:
So it comes before hte suggestion to add-braces on an 'if' :)

Reply:
@mavasani Since [ExtensionOrderAttribute](https://learn.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.extensionorderattribute?view=roslyn-dotnet-4.3.0) is public, can't we use it to fix https://github.com/dotnet/roslyn/issues/47766?